assets/exam css

```diff
@@ -0,0 +1,3 @@
+body{
+  background;
```

Comment:
This CSS 'background' property has no value ( property: value; ).

Perhaps you were playing around with something and meant to remove it?